docs/setup.md

```diff
@@ -23,7 +23,7 @@ Complicated to start, but easy to develop from.
 ```bash
 # install the database you want to use, samson needs mysql, postgresql, or sqlite
 sudo apt-get install mysql-dev pg-dev nodejs
-brew install postgresql sqlite mysql
+brew install postgresql sqlite mysql nodejs
```

Comment:
why does it need nodejs?

Reply:
So, it is a dependency for new rails?

Reply:
I know Rails has a JS runtime dependency but I'm assuming since we don't have `execjs`, `nodejs`, or `therubyracer` in the Gemfile, Rails tries to default to system `nodejs` (which probably many has it installed for other reasons).